react-frontend/src/login/PreLoginDashboardComponent.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import PeerlyTextAndLogo from "shared-components/peerly-logo/PeerlyTextAndLogo";
+import InformativeTextComponent from "login/InformativeTextComponent";
+import LoginImageComponent from "login/LoginImageComponent";
+
+const Wrapper = styled.div`
+  height: 100vh;
+  background-color: ${({ theme }) =>
+    theme === "dark" ? "var(--atomic)" : "var(--white)"};
+`;
+
+const PreLoginDashboardComponent = ({ theme, className }) => (
```

Comment:
where do you think we are going to use this component? will this component have a container? is there any state management required for this? why would the parent of this component decide the theme? this isn’t a shared component, also parent component decides the style (classname)? of presentational (NOT shared) component?